CMakeLists.txt

```diff
@@ -1996,6 +2018,10 @@ add_library(${LOVE_LIB_NAME} SHARED ${LOVE_LIB_SRC} ${LOVE_RC})
 set_target_properties(${LOVE_LIB_NAME} PROPERTIES C_VISIBILITY_PRESET hidden CXX_VISIBILITY_PRESET hidden VISIBILITY_INLINES_HIDDEN ON)
 target_link_libraries(${LOVE_LIB_NAME} ${LOVE_LINK_LIBRARIES} ${LOVE_3P})
 
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
+	set_target_properties(${LOVE_LIB_NAME} PROPERTIES LIBRARY_OUTPUT_NAME "${LOVE_EXE_NAME}-${LOVE_VERSION_STR}")

```

Comment:
This also seems vaguely related to the `set(LOVE_LIB_NAME)` for android. Maybe `LOVE_EXE_NAME` and `LOVE_LIB_NAME` should just set the `OUTPUT_NAME` properties, rather than actually being the target names?

Reply:
Good idea, although this need extensive testing both on Windows and Android.